src/plugins/unified_search/public/filter_bar/filter_editor/index.tsx

```diff
@@ -199,43 +199,41 @@ class FilterEditorUI extends Component<Props, State> {
     );
   }
 
-  private renderIndexPatternInput() {
+  private renderDataViewInput() {
     if (
-      this.props.indexPatterns.length <= 1 &&
-      this.props.indexPatterns.find(
-        (indexPattern) => indexPattern === this.getIndexPatternFromFilter()
-      )
+      this.props.dataViews.length <= 1 &&
+      this.props.dataViews.find((dataView) => dataView === this.getDataViewFromFilter())
     ) {
       /**
-       * Don't render the index pattern selector if there's just one \ zero index patterns
-       * and if the index pattern the filter was LOADED with is in the indexPatterns list.
+       * Don't render the data view selector if there's just one \ zero data views
+       * and if the data view the filter was LOADED with is in the dataViews list.
        **/
 
       return '';
     }
-    const { selectedIndexPattern } = this.state;
+    const { selectedDataView } = this.state;
     return (
       <>
         <EuiFormRow
           fullWidth
           label={this.props.intl.formatMessage({
-            id: 'unifiedSearch.filter.filterEditor.dateViewSelectLabel',
-            defaultMessage: 'Data view',
+            id: 'unifiedSearch.filter.filterEditor.dataViewSelectLabel',
+            defaultMessage: 'Index Pattern',
```

Comment:
Why have we changed this to Index Pattern? Data view is the correct text here, right?